src/core/MOM_open_boundary.F90

```diff
@@ -528,13 +528,10 @@ subroutine open_boundary_config(G, US, param_file, OBC)
       OBC%add_tide_constituents = .false.
     endif
 
-    call get_param(param_file, mdl, "DEBUG", OBC%debug, default=.false.)
-    call get_param(param_file, mdl, "DEBUG_OBC", debug_OBC, default=.false.)
-    if (debug_OBC .or. OBC%debug) &
-      call log_param(param_file, mdl, "DEBUG_OBC", debug_OBC, &
-                 "If true, do additional calls to help debug the performance "//&
-                 "of the open boundary condition code.", default=.false., &
-                 debuggingParam=.true.)
```

Comment:
We probably want to retain the message from the `log_param(..., "DEBUG_OBC", ...)` call, as well as the `debugginParam=.true.` argument in the new `get_param(..., "DEBUG_OBC", ...)` to retain the message about this run-time option.